manageiq-operator/go.mod

```diff
@@ -4,12 +4,12 @@ go 1.20
 
 require (
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.27.6
-	github.com/openshift/api v3.9.0+incompatible
-	k8s.io/api v0.26.3
-	k8s.io/apimachinery v0.26.3
-	k8s.io/client-go v0.26.3
-	sigs.k8s.io/controller-runtime v0.14.6
+	github.com/onsi/gomega v1.27.8
+	github.com/openshift/api v0.0.0-20230705144233-e28cd4dd28a8
```

Comment:
? Did this downgrade?  Or does this just mean "unversioned"?

Reply:
Unfortunately they don't seem to have real releases, so this is latest